### PR TITLE
Pratiksha work

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html]: check indentation head is a child tag of html tag make this change in every other assignments

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html]: remove extra line

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html]: give proper title change in every other assignment

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+
+</head>
+<body>
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html]: check indentation body is a child tag of html tag make this change in every other assignments

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    <div class="container">
+        <nav aria-label="breadcrumb">
+            <ol class="breadcrumb m-3">
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html]: here 4 spaces from parent tag. its not consistent

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+              <li class="breadcrumb-item"><a href="#">Home</a></li>
+              <li class="breadcrumb-item"><a href="#">Employee</a></li>
+              <li class="breadcrumb-item active" aria-current="page">Employee Details
+              </li>
+            </ol>
+          </nav>
+          <h1>List of Employee<span class="badge badge-secondary">New</span></h1>
+          <table class="table">
+            <thead class="thead-light">
+              <tr>
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html]: here 2 spaces from parent tag not consistent please check 
required 4 spaces from parent tag to child tag 
check this in each and every file

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+            </tbody>
+          </table>
+    </div>
+
+
+
+
+
+
+
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html]: remove extra lines

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0"
+        crossorigin="anonymous"></script>
+</body>
+</html>
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html]: add extra line at the end of file make this change in every file

File: Pratiksha/Javascript/Assignment2/Q1.html [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+      } else if (trdNo > fstNo && trdNo > secNo && trdNo > frtNo && trdNo > fifNo) {
+        document.getElementById("largest")
+      } else if (frtNo > fstNo && frtNo > trdNo && frtNo > secNo && frtNo > fifNo) {
+        document.getElementById("largest")
+      } else {
+        document.getElementById("largest")  
+      }
+    }
+    </script>
+    <script
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment2/Q1.html]: use single script tag

File: Pratiksha/Javascript/Assignment2/Q2.html [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+      >
+        Grade
+      </button>
+    </form>
+    <p id="demo"></p>
+    <p id="demo1"></p>
+    <script>
+      var c;
+      function calculate() {
+        var x, y, z, a, b;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment2/Q2.html]: change variable names in every file don't use a, b, c as a variable name, variable name should be related to functionality

File: Pratiksha/Javascript/Assignment3/Q1.html [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+      </button>
+    </form>
+    <p id="demo"></p>
+    <script>
+      year= document.getElementById("val1").value;
+      function leap() {
+        if (year % 4 == 0) {
+          document.getElementById("demo").innerText = `Leap year`;
+        } else {
+          document.getElementById("demo").innerText = `  Not Leap year`;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q1.html]: remove extra space present before Not leap year

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prime</title>
+</head>
+<body>
+   
+    
+
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:12]: remove extra lines

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 13]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:3]: check indentation

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by pratikshaghule — COMMENTED** [review]
Mam, review it plz

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+    />
+  </head>
+  <body>
+    <div class="container">
+      <nav aria-label="breadcrumb">
+         <ol class="breadcrumb m-3">
+            <li class="breadcrumb-item"><a href="#">Home</a></li>
+            <li class="breadcrumb-item"><a href="#">Employee</a></li>
+            <li class="breadcrumb-item active" aria-current="page">
+            Employee Details
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html:22]: check indentation need 4 spaces here

File: Pratiksha/Javascript/Assignment2/Q2.html [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+        type="button"
+        class="ml-4 btn btn-primary center"
+      >
+        Grade
+      </button>
+    </form>
+    <p id="demo"></p>
+    <p id="demo1"></p>
+    <script>
+      var c;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment2/Q2.html]: change variable name

File: Pratiksha/Javascript/Assignment3/Q2 .html (hunk 13 above)
Comment (KomalShinde813) [reply Pratiksha/Javascript/Assignment3/Q2 .html:3]: indentation change not done here

File: Pratiksha/Javascript/Assignment3/Q2 .html (hunk 12 above)
Comment (KomalShinde813) [reply Pratiksha/Javascript/Assignment3/Q2 .html:12]: check this file again

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+                <br><br>
+                  
+                <input type="button" value="Check" onClick="p()">
+                <br>
+            </form>
+        <p id="demo"></p>
+        <script>
+        function p() {
+              
+            var n, i, flag = true;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:32]: change variable names

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 17]
```diff
@@ -0,0 +1,52 @@
...
+                <input type="text" name=n value="">
+                  
+                <br><br>
+                  
+                <input type="button" value="Check" onClick="p()">
+                <br>
+            </form>
+        <p id="demo"></p>
+        <script>
+        function p() {
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:30]: change function name

File: Pratiksha/Javascript/Assignment4/Q3.html [hunk 18]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q3.html:3]: please check all files and do the changes

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 19]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html:3]: check indentation

File: Pratiksha/Javascript/Assignment4/Q1.html [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+      if (arm == num) {
+        alert("Armstrong number");
+      }
+      else {
+        alert("Not Armstrong number");
+      }
+    }
+  </script>
+  </body>
+</html>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q1.html:32]: add extra line at the end of file

File: Pratiksha/Javascript/Assignment4/Q1.html [hunk 21]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Armstrong</title>
+</head>
+<body>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q1.html:9]: check indentation properly

File: Pratiksha/Javascript/Assignment3/Q_2.html [hunk 22]
```diff
@@ -0,0 +1,37 @@
...
+      Enter the number:
+      <input type="text" name="n" value="" />
+      <br /><br />
+      <input type="button" value="Check" onClick="primeornot()" />
+      <br />
+    </form>
+    <p id="demo"></p>
+    <script>
+      function primeornot() {
+        var n,
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q_2.html]: change variable names

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**